spec/services/block_domain_from_account_service_spec.rb

```diff
@@ -11,9 +11,9 @@
     expect(alice.domain_blocking?('evil.org')).to be true
   end
 
-  it 'purge followers from blocked domain' do
```

Comment:
No need of spec for behavior without code. Preserving such will keep spec growing.

Reply:
Well, ok, but it confirms the behavior is intentional, as opposed to leading it unspecified

Reply:
Updated to match the requested changes, although I kinda disagree, as mentioned earlier.

Reply:
I'm pretty sure that anyone who found this change is undesired can figure out that follow relations from the blocking users are preserved. It is fine in the case.